src/components/AddItemForm.tsx

```diff
@@ -1,22 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-function AddItemForm() {
+function AddItemForm(props: any) {
+  const { setDescription, description } = props;
   const domInputId = 'new-todo-input';
+  const [captureInput, setCaptureInput] = useState<string>();
+
+  const handleInputText = () => {
+    const listDescription = [...description];
+    listDescription.push(captureInput);
```

Comment:
Estamos usando o spread operator errado.
- https://www.samanthaming.com/tidbits/87-5-ways-to-append-item-to-array/

Recomendo:
- joga todos os items anteriores
- adiciona o novo item no final

```suggestion
    const listDescription = [
      ...description,
      captureInput,
    ];
```